### PR TITLE
Updated some bits and added awscli

### PR DESCRIPTION
**Pull request by gavinmcnair**
I've updated some of the versions.

The scripts should really be updated but that can wait for another day.

### CODE REVIEW

**Comment by swade1987** [comment]
LGTM thanks @gavinmcnair 👍

**Comment by swade1987** [comment]
@gavinmcnair might be worth updating https://github.com/swade1987/kubernetes-toolkit/blob/master/Makefile#L5 as well.

**Comment by gavinmcnair** [comment]
> @gavinmcnair might be worth updating https://github.com/swade1987/kubernetes-toolkit/blob/master/Makefile#L5 as well.
> 
> This is used as the image tag so might as well upgrade to the latest version.

I've updated this again. Thanks.

File: Makefile [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # Project build information
 #------------------------------------------------------------------
 PROJNAME := kubernetes-toolkit
-KUBERNETES_VERSION := 1.18.14
+KUBERNETES_VERSION := 1.21.00
```
Comment (swade1987) [inline Makefile:5]: I would either go with the latest `1.20` release which is `1.20.14`  or the most recent release altogether which is `1.23.3`

Comment (gavinmcnair) [reply Makefile]: I went wit 1.21.8

**Pull request merged**